<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Route App</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,2">
    <item>
     <widget class="QGroupBox" name="routeGroup">
      <property name="title">
       <string>Route</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="routeTableView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="routesActionsLayout">
         <item>
          <widget class="QPushButton" name="addRoutePushButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeRoutePushButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="pointsGroup">
      <property name="title">
       <string>Points</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="pointTableView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="pointsActionsLayout">
         <item>
          <widget class="QPushButton" name="addPointPushButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removePointPushButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="editGroup">
      <property name="title">
       <string>Edit</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="latitudeLabel">
         <property name="text">
          <string>Latitude: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="routeNameLabel">
         <property name="text">
          <string>Route Name: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="longitudeLabel">
         <property name="text">
          <string>Longitude: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="longitudeLine"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="routeNameLine"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="latitudeLine"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="heightLabel">
         <property name="text">
          <string>Height: </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="heightLine"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionHeight_Map"/>
   </widget>
   <widget class="QMenu" name="menuPolyline">
    <property name="title">
     <string>Polyline</string>
    </property>
    <addaction name="actionCreateRoute"/>
    <addaction name="actionUpdatePolyline"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
    <addaction name="actionPlugin_table"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPolyline"/>
   <addaction name="menuPlugins"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCreateRoute">
   <property name="text">
    <string>Create route</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUpdatePolyline">
   <property name="text">
    <string>Save route</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionHeight_Map">
   <property name="text">
    <string>Height Map</string>
   </property>
   <property name="shortcut">
    <string>Shift+Q</string>
   </property>
  </action>
  <action name="actionPlugin_table">
   <property name="text">
    <string>Plugin Table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
